web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/aksjonspunkt/AksjonspunktDto.java

```diff
@@ -3,27 +3,31 @@
 import java.time.LocalDateTime;
 import java.util.Set;
 
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import jakarta.validation.constraints.NotNull;
+
 import no.nav.foreldrepenger.behandlingslager.behandling.aksjonspunkt.AksjonspunktDefinisjon;
 import no.nav.foreldrepenger.behandlingslager.behandling.aksjonspunkt.AksjonspunktStatus;
 import no.nav.foreldrepenger.behandlingslager.behandling.aksjonspunkt.AksjonspunktType;
 import no.nav.foreldrepenger.behandlingslager.behandling.aksjonspunkt.VurderÅrsak;
 import no.nav.foreldrepenger.behandlingslager.behandling.vilkår.VilkårType;
 
 public class AksjonspunktDto {
-    private AksjonspunktDefinisjon definisjon;
-    private AksjonspunktStatus status;
-    private String begrunnelse;
-    private VilkårType vilkarType;
-    private Boolean toTrinnsBehandling;
-    private Boolean toTrinnsBehandlingGodkjent;
-    private Set<VurderÅrsak> vurderPaNyttArsaker;
-    private String besluttersBegrunnelse;
-    private AksjonspunktType aksjonspunktType;
-    private Boolean kanLoses;
-    private Boolean erAktivt;
-    private LocalDateTime fristTid;
-    private LocalDateTime endretTidspunkt;
-    private String endretAv;
+    @NotNull private AksjonspunktDefinisjon definisjon;
+    @NotNull private AksjonspunktStatus status;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private String begrunnelse;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private VilkårType vilkarType;
+    @NotNull private Boolean toTrinnsBehandling;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private Boolean toTrinnsBehandlingGodkjent;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private Set<VurderÅrsak> vurderPaNyttArsaker;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private String besluttersBegrunnelse;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private AksjonspunktType aksjonspunktType;
+    @NotNull private Boolean kanLoses;
+    @NotNull private Boolean erAktivt;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private LocalDateTime fristTid;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private LocalDateTime endretTidspunkt;
+    @Schema(nullable = true, requiredMode = Schema.RequiredMode.REQUIRED) private String endretAv;
```

Comment:
Hvorfor alle REQUIRED? Kan man ikke ha en default for alt / klassen og markere der det er avvik.
Går på lesbarhet